.github/ISSUE_TEMPLATE/docs_issue.md

```diff
@@ -1,20 +1,7 @@
 ---
 name: Docs Issue
-about: Report issues or request new content in the Crossplane documentation.
+about: Open docs issues at https://github.com/crossplane/docs/issues
 labels: ["docs"]
 ---
 
-<!-- If it's a bug tell us where you found it -->
-## Docs URL
-<!-- Provide the URL to the page that has a bug. Use the nearest anchor if possible -->
-<!-- for example 
-https://crossplane.io/docs/master/getting-started/install-configure.html#start-with-upstream-crossplane
--->
-
-<!-- This is only needed for bugs -->
-## Affected versions
-<!-- If you can, check if this problem exists in the other versions -->
-
-## What's wrong?
-<!-- tell us the error or problem you found in the documentation -->
-<!-- for content requests tell us what content you'd like to see created -->
+Report issues with the Crossplane documentation in the [docs repository](https://github.com/crossplane/docs/issues). 
```

Comment:
It looks like you can actually just make a link. Happy to open a subsequent PR to keep this moving along.

For example see https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/config.yml#L1 and https://github.com/kubernetes/kubernetes/issues/new/choose